### PR TITLE
EFU-891 Create component: Templates Card

### PR DESCRIPTION
**Pull request by mheob**
image

### CODE REVIEW

File: packages/cards/src/TemplateCard.tsx [hunk 1]
```diff
@@ -52,7 +63,7 @@ export const TemplateCard = ({
         zIndex="hide"
       />
 
-      <Box pos="absolute" top="1" right="1" zIndex="docked">
+      <Box pos="absolute" top="1" title="options" right="1" zIndex={theme.zIndices.docked + 1}>
```
Comment (mheob) [inline packages/cards/src/TemplateCard.tsx:66]: Why is the `title` hardcoded?
If we do that, which is definitely fine, we should just prepare ourselves directly to be multilingual.

File: packages/cards/src/TemplateCard.tsx [hunk 2]
```diff
@@ -80,10 +94,10 @@ export const TemplateCard = ({
 
         {views && (
           <HStack color="gray.700" data-testid="views">
-            <Text as="span" fontSize="sm">
+            <Text as="span" zIndex="docked" title={views} fontSize="sm" isTruncated maxWidth={145}>
               {views}
             </Text>
-            <Icon as={IoStatsChartSharp} w="3" h="3" />
+            <Icon as={IoStatsChartSharp} title="survey stats" zIndex="docked" w="3" h="3" />
```
Comment (mheob) [inline packages/cards/src/TemplateCard.tsx:100]: Same as above (hardcoded, multilingual).

**Review by ef-jgiannelli — APPROVED** [review]
nice work @mheob !
as requested the fixes in my own commits ;)

File: packages/cards/stories/TemplateCard.stories.tsx [hunk 3]
```diff
@@ -17,7 +17,7 @@ export default {
     title: 'Demo Survey (Satisfaction Ice Cream)',
     imageSrc:
       'https://images.unsplash.com/photo-1534551767192-78b8dd45b51b?fit=crop&w=220&h=202&q=60',
-    views: '213',
+    views: 213,
```
Comment (mheob) [inline packages/cards/stories/TemplateCard.stories.tsx:20]: Do the views actually come back as numbers from GraphQL?
If so, we should also directly adjust the type definition in https://github.com/easyfeedback/RCL/blob/185bd0e8d0b4f3c0e36567d86d00dd3d4a22817d/packages/cards/src/TemplateCard.tsx#L25

File: packages/cards/stories/TemplateCard.stories.tsx [hunk 4]
```diff
@@ -36,5 +36,13 @@ export const Default = Template.bind({})
...
-  state: 'planned',
+  state: 'initialized',
+}
+
+export const Limits = Template.bind({})
+Limits.args = {
+  title:
+    'Survey about something with users from the Application of state and democracy - inspirated by rainbows and flowers',
+  state: 'closed',
+  views: 12345678901234567890,
```
Comment (mheob) [inline packages/cards/stories/TemplateCard.stories.tsx:47]: Same as above (type definition).

**Comment by mheob** [comment]
I have added some thoughts that I have found. They are not blockers, but wanted to mention them.

File: packages/cards/src/TemplateCard.tsx (hunk 1 above)
Comment (ef-jgiannelli) [reply packages/cards/src/TemplateCard.tsx:66]: well I didnt saw a i18n or similar in the codebase, but yes we need locales!
in StateDot.tsx the state also declares `title` and is hardcoded
any suggestions?

File: packages/cards/stories/TemplateCard.stories.tsx (hunk 3 above)
Comment (ef-jgiannelli) [reply packages/cards/stories/TemplateCard.stories.tsx:20]: see last commit! :dancers:

File: packages/cards/src/TemplateCard.tsx (hunk 1 above)
Comment (mheob) [reply packages/cards/src/TemplateCard.tsx:66]: Yes, correct. I had only added the cases as placeholders. So I didn't really think about it there.
And no. At the moment, I don`t have any proposal. I will add `TODO` statements and open a Jira task for it. We can think about it at a later stage.

**Pull request merged**